types/tx.go

```diff
@@ -138,22 +138,10 @@ func TxProofFromProto(pb tmproto.TxProof) (TxProof, error) {
 	return pbtp, nil
 }
 
-// ComputeProtoOverheadForTxs wraps the transactions in tmproto.Data{} and calculates the size.
+// ComputeProtoSizeForTxs wraps the transactions in tmproto.Data{} and calculates the size.
 // https://developers.google.com/protocol-buffers/docs/encoding
-func ComputeProtoSizeForTxs(txs []Tx, tx Tx) int64 {
-	tt := make(Txs, len(txs)+1)
-
-	tt = append(tt, txs...)
-	tt = append(tt, tx)
-	data := Data{Txs: tt}
+func ComputeProtoSizeForTxs(txs []Tx) int64 {
+	data := Data{Txs: txs}
 	pdData := data.ToProto()
 	return int64(pdData.Size())
```

Comment:
Just code golfing here, but this possibly reduces to `return int64(Data{Txs: txs}.ToProto().Size())`, unless any of the methods have pointer receivers.

Reply:
Actually it is possible if I remove `nullable=false`